### PR TITLE
Apple clone branch

### PR DESCRIPTION
**Pull request by camfilho**
Branch ready to be sent to be reviewed by the TSAs.

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
Hi Carlos 👋
Here are a few changes to make in other to get this project approved.
- #### **[REQUIRED]** NAVBAR🙂
1. It should be made of nine different buttons with the Apple logo on the leftmost section and a search field on the rightmost section
> As it stands now the search has no input field and icon.
>  your nav links should be wrapped in anchor tag instead 
>  You can achieve the button effect by giving hover effect to the ```anchor``` tags and giving ```borders``` 
2. The navbar ought to have a background gradient implementation. [read](https://www.w3schools.com/css/css3_gradients.asp)

- #### **[REQUIRED]** project implementation🙂
1. Each Image in ```main``` have to be wrapped in an anchor tag so they redirect user upon clicking it
> images are added to a page by using ```img``` tag
2. Set the width of the ```.wrapper``` to 100%, to have filled the whole screen.

# Kindly make those changes and submit for review again
Happy coding

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                <a href="#">
+                    <img src="https://archive.fo/UW4oR/d7dd20c4365117e1cc1b3c2492508e9ba842e5d5.png" alt="headline 1">
+                    <img src="https://archive.fo/UW4oR/98e684c02dfa6d288709a583e375cfbdea4b3f7d.png" alt="headline 1">
+                    <img src="./images/headline_edited.png" alt="headline 1">
+               </a>
+            </header>
+        </div>
+        <main class="row">
+            <div class="col-4">
+                <div class="img-1"></div>
```
Comment (Forison) [inline index.html]: # use `````` tag for the image and wrap ```<a>``` around each image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            </div>
+            <div class="col-4">
+                <div class="img-2"></div>
+            </div>
+            <div class="col-4">
+                <a href="#" class="img-3"></a>
```
Comment (Forison) [inline index.html]: # use `````` tag for the image and wrap ```<a>``` around each image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                    <img src="./images/headline_edited.png" alt="headline 1">
+               </a>
+            </header>
+        </div>
+        <main class="row">
+            <div class="col-4">
+                <div class="img-1"></div>
+            </div>
+            <div class="col-4">
+                <div class="img-2"></div>
```
Comment (Forison) [inline index.html]: # use `````` tag for the image and wrap ```<a>``` around each image ```<a>  </a>```

**Review by whiz25 — APPROVED** [review]
## Awesome job :+1: @camfilho 
You did good. All checks have passed and all requirements met.
Please go ahead and merge. I wish you luck in your next projects.

**Pull request merged**